qiskit/circuit/library/standard_gates/u1.py

```diff
@@ -332,6 +348,22 @@ class MCU1Gate(ControlledGate):
     This is a diagonal and symmetric gate that induces a
     phase on the state of the target qubit, depending on the state of the control qubits.
 
+    .. warning::
+
+       This gate is deprecated. Instead, the following replacements should be used
+
+       .. math::
+
+           MCU1(\lambda) = MCP(\lambda)= MCU(0,0,\lambda,0)
+
+       .. code-block:: python
+
+          circuit = QuantumCircuit(2)
+          circuit.mcp(lambda, 0, 1) # or circuit.mcu(0, 0, lambda, 0, 0, 1)
```

Comment:
Is there a `QuantumCircuit.mcu` method? 🤔

Reply:
nope